kokoro/linux/aarch64/protoc_crosscompile_aarch64.sh

```diff
@@ -6,6 +6,3 @@ set -ex
 
 cmake -DCMAKE_POSITION_INDEPENDENT_CODE=ON -Dprotobuf_WITH_ZLIB=0 .
```

Comment:
It's not, this test is specifically for aarch64.  The reason we use CMake is because I had trouble getting Bazel to work in an earlier PR.  I have a follow up task to replace these with a set of more general cross-compilation tests that use bazel